lib/app.rb

```diff
@@ -50,9 +49,10 @@ class App < Sinatra::Base
     use Routes::Comments
     use Routes::Nits
     use Routes::Teams
+    use Routes::Sessions
```

Comment:
Let me check on that--technically only Teams was having an issue with it, but when we were first figuring this out with the language info pages, only moving it to the bottom made them visible without login.